### PR TITLE
add user to Slug API

### PR DESCRIPTION
**Pull request by rankitranjan**
BLCR-4068

### CODE REVIEW

File: paths/unique_identifiers.yaml [hunk 1]
```diff
@@ -5,6 +5,15 @@ get:
   description: |
     Get uuid for given model currently supported models are Tag, StaticPage, Producttest, PrivateGroup, Category, Question, Feedback, Article, Faq, Tutorial, ProductReview, ProducttestApplication, WidgetContainer
   parameters:
+    - name: user
+      in: body
```
Comment (Xtimesx) [inline paths/unique_identifiers.yaml:9]: parameters in the body are marked up differently in the openapi version 2
do you mean query?

Comment (rankitranjan) [reply paths/unique_identifiers.yaml:9]: nope in body.

Comment (Xtimesx) [reply paths/unique_identifiers.yaml:9]: but then it shouldn't be get. GET is not standardized to parse GET bodies, should such exist

Comment (rankitranjan) [reply paths/unique_identifiers.yaml:9]: ah I overlooked it 
it is actually query params, 
http://localhost:3000/api/2.1/unique_identifiers?tag[]=enrollee&question[]=asdasdasdadasd&static_page[]=dasdas&user[]=testadmin

**Review by Xtimesx — APPROVED** [review]

**Pull request merged**